packages/documentation/docs/getting-started/installation/installing-a-gateway/rundown-or-newsroom-system-connection/installing-sofie-with-google-spreadsheet-support.md

```diff
@@ -4,15 +4,15 @@ The Spreadsheet Gateway is an application for piping data between Sofie Core and
 
 ### Example Blueprints for Spreadsheet Gateway
 
-To begin with, you will need to install a set of Blueprints that can handle the data being sent from the _Gateway_ to _Core_. Download the ZIP file containing the three files you need from the [Spreadsheet Gateway's GitHub Repository](https://github.com/SuperFlyTV/sofie-blueprints-spreadsheet/releases). It is recommended to choose the newest release but, an older _Core_ version may require a different Blueprint version. The _Rundown page_ will warn you about any issue and display the desired versions. 
+To begin with, you will need to install a set of Blueprints that can handle the data being sent from the _Gateway_ to _Core_. Download the `bundle.json` file containing the blueprints you need from the [Demo Blueprints GitHub Repository](https://github.com/SuperFlyTV/sofie-demo-blueprints/releases). It is recommended to choose the newest release but, an older _Core_ version may require a different Blueprint version. The _Rundown page_ will warn you about any issue and display the desired versions.
```

Comment:
I thought you found this `bundle.json` doesn't work?

Reply:
Good catch, fixed.